src/app/leaves/components/LeaveActionRender.tsx

```diff
@@ -54,7 +81,16 @@ const LeaveTableActionRender = ({ leaveRequestDetails }: Props) => {
       {
         title: "Delete",
         id: 3,
-        onClick: handleDeleteRequest,
+        onClick: () => {
+          if (requestId !== undefined) {
+            handleDeleteRequest(requestId);
```

Comment:
can you explain why we need this condition because when we click on any leave that leave must have id it is impossible that leave will not have id ?

Reply:
There is a possibility that `Id` could be undefined, which is why I added a check.